### PR TITLE
Remove rhel7 stuff from the develop (ubi8) branch

### PR DESCRIPTION
**Pull request by jmtd**
Part of re-organising described in #307.

### CODE REVIEW

**Comment by jerboaa** [comment]
@jmtd `dejavu-sans-mono-fonts` apparently is in the UBI 8 set now, but tests seem to complain about it. Could you please fix this (another PR?) so that we have passing checks?

File: README.adoc [hunk 1]
```diff
@@ -11,21 +12,17 @@ link:https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI[UBI
...
- * link:https://catalog.redhat.com/software/containers/redhat-openjdk-18/openjdk18-openshift/58ada5701fbe981673cd6b10[redhat-openjdk-18/openjdk18-openshift]
-  * link:https://catalog.redhat.com/software/containers/openjdk/openjdk-11-rhel7/5bf57185dd19c775cddc4ce5[openjdk/openjdk-11-rhel7]
+ * link:https://catalog.redhat.com/software/containers/ubi8/openjdk-17/618bdbf34ae3739687568813[ubi8/openjdk-17]
 
 ## How to build the images
 
 You need to https://cekit.readthedocs.io/en/develop/installation.html[install Cekit] to build these images.
 
-These sources are prepared and tested for Cekit 3.6.0.
+These sources are prepared and tested for Cekit 4.1.1.
```
Comment (jerboaa) [inline README.adoc:21]: Nit: Latest is `4.3.0`.

**Review by jerboaa — APPROVED** [review]
LGTM. Passing tests would be nice ;-)

File: README.adoc (hunk 1 above)
Comment (jmtd) [reply README.adoc:21]: > Nit: Latest is 4.3.0.

True. I haven't tried that version yet!

**Comment by jmtd** [comment]
> @jmtd `dejavu-sans-mono-fonts` apparently is in the UBI 8 set now, but tests seem to complain about it. Could you please fix this (another PR?) so that we have passing checks?

Sure, filed  https://issues.redhat.com/browse/OPENJDK-986 to track

**Pull request merged**